api/cds.proto

```diff
@@ -272,4 +272,28 @@ message Cluster {
     repeated LbSubsetSelector subset_selectors = 3;
   }
   LbSubsetConfig lb_subset_config = 22;
+  
+  message RingHashLbConfig {
+    // The hash function used to build the consistent hashing ring by hashing
+    // each server's address.
+    enum HashFunction {
+      // Standard library
+      STL = 0;
```

Comment:
It's unfortunate that we need to add this, but I guess we do for back-compat. Is there any way around this? probably not.

Reply:
I don't fully understand the rationale here for adding this to v2 (why not make it xxHash only there?). Could this be under a `deprecated_v1` if just used for v1 translation?

Reply:
Interesting thought, but now that I consder it I think we should allow moving to v2 APIs without having to potentially rehash your ring_lb. I could go either way though.

Reply:
I would put this in deprecated_v1 section per @htuch and make people use that if they are migrating.